packages/react/src/components/pickers/BasePicker.types.ts

```diff
@@ -173,6 +173,12 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    */
   removeButtonAriaLabel?: string;
 
+  /**
+   * The text that will be announced when a suggestion is removed. A default value is only provided for English.
+   * @default "removed {0}"
```

Comment:
This will fix the API file error
```suggestion
   * @default `"removed {0}"`
```